plugins/wolframalpha.py

```diff
@@ -47,4 +47,4 @@ def unicode_sub(match):
     if not ret:
         return 'no results'
 
-    return ret
+    return ret.replace('Result: ', '', 1)
```

Comment:
Do this on line 28-- `if title == 'Result': pod_texts.append('|'.join(results)) else` etc